validation-test/Sema/SwiftUI/rdar84580119.swift

```diff
@@ -14,8 +14,9 @@ extension EnvironmentValues {
   var myHorizontalAlignment: AlignmentID? {
     get { fatalError() }
     set { self[\.MyHorizontalAlignmentEnvironmentKey.self] = newValue }
-    // expected-error@-1 {{subscript 'subscript(_:)' requires that 'any AlignmentID' be a class type}}
-    // expected-error@-2 {{cannot infer key path type from context; consider explicitly specifying a root type}}
+    // FIXME: error lost {{subscript 'subscript(_:)' requires that 'any AlignmentID' be a class type}}
```

Comment:
I don't actually think we want that diagnostic anyway, it was because we picked a subscript overload that expected a metatype argument and was read-only, so it seems pretty unlikely that's what was expected here